monkey/monkey_island/cc/ui/src/components/report-components/ransomware/AttackSection.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {ReactElement} from 'react';
+import {FileEncryptionTable, TableRow} from './FileEncryptionTable';
+import NumberedReportSection from './NumberedReportSection';
+
+const ATTACK_DESCRIPTION = 'After the attacker or malware has propagated through your network, \
+                            your data is at risk on any machine the attacker can access. It can be \
+                            encrypted and held for ransomware, exfiltrated, or manipulated in \
+                            whatever way the attacker chooses.'
```

Comment:
The wording of this can be simplified and in case marketing misses it I leave my suggestion here:
`After the attacker has propagated through your network, he will try to exfiltrate and encrypt all of the files he got access to.`

The text in `LateralMovement.tsx` needs to be reworked as well, as it speaks about encryption in "Lateral Movement" step.